tests/fast-integration/Makefile

```diff
@@ -3,7 +3,6 @@ ci:
 	npm install
 	npm test
 	npm run test-monitoring
-	npm run test-eventing
```

Comment:
Please document in this file on top of L45 (or if there is some README for fast integration tests), how one can run this make target outside of Prow. What env variable is required to read the EMS secret, and what should be the content of the file.

Reply:
added in readme.md